Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get update && \
     unzip -q ./activator.zip && \
     cd collins && \
     java -version 2>&1 && \
```

Comment:
Yeah, I think that was just added for debugging purposes.  @byxorna would be able to tell us for sure though 😄